src/Repository/CitizenProjectMembershipRepository.php

```diff
@@ -147,6 +147,17 @@ public function findPriviledgedMembers(string $citizenProjectUuid, array $privil
         return $this->createAdherentCollection($query);
     }
 
+    public function findFollowers(string $citizenProjectUuid, bool $includeAdministrators = true): AdherentCollection
+    {
+        $privileges = [CitizenProjectMembership::CITIZEN_PROJECT_FOLLOWER];
+
+        if ($includeAdministrators) {
+            $privileges = array_merge($privileges, [CitizenProjectMembership::CITIZEN_PROJECT_ADMINISTRATOR]);
```

Comment:
Simpler to do :

`$privileges[] = CitizenProjectMembership::CITIZEN_PROJECT_ADMINISTRATOR;`